composer.json

```diff
@@ -25,13 +25,14 @@
     ],
     "autoload": {
         "psr-4": {
-            "Dcentrica\\Omnipay\\BTCPayServer\\": "src/"
+            "Omnipay\\BTCPayServer\\": "src/"
         }
     },
     "require": {
         "php": ">=7.1",
         "omnipay/common": "~3.0",
-        "bitpay/php-client": "dev-master"
+        "bitpay/php-client": "dev-master",
+        "btcpayserver/btcpayserver-php-client": "~1.0.0"
```

Comment:
We'll stick with `bitpay/php-client` - `btcpayserver/btcpayserver-php-client` looks to be buggy as hell.